src/adapters/data_models/shortcuts_model/shortcuts_proxy_model.cpp

```diff
@@ -30,19 +30,25 @@ bool ShortcutsProxyModel::lessThan(const QModelIndex& left,
     QString rightName = sourceModel()->data(right, ShortcutRole).toString();
 
     // Sort alphabetically if no filter string is set
-    if(m_filterString.isEmpty())
+    if(!m_filterScorer)
         return leftName < rightName;
 
     // Sort by the highest similarity if filter string is set
-    auto lSimilarity = string_utils::fuzzCompare(leftName, m_filterString);
-    auto rSimilarity = string_utils::fuzzCompare(rightName, m_filterString);
+    auto lSimilarity = string_utils::substringCompare(leftName, m_filterString);
+    if(lSimilarity == 0.0)
```

Comment:
not really happy with this compare. It would work here, since we explicitly set it to 0 in there, but == on float is always sketchy. Probably should be done differently.

Reply:
Good point, there is a Qt function for fuzzy comparing floats, I'll use those